tests/test_web_comprehensive.py

```
@@ -0,0 +1,612 @@
+    import logging
+
+    return logging.getLogger("test")
+
+
+@pytest.fixture
+def mock_config():
+    """Create a mock config."""
+    config = MagicMock(spec=AppConfig)
+    config.get.side_effect = lambda s, k, **kw: {
+        ("Watcher", "check_interval"): 60,
+        ("Paths", "all_entries_log"): "/tmp/test_entries.csv",
```

Ruff (0.14.14)

[error] Probable insecure usage of temporary file or directory: "/tmp/test_entries.csv"

(S108)